### DISCUSSION TITLE
Model Database

### DISCUSSION THREAD

**Discussion by Doomerdinger** [dfki-ric/phobos#337] [Q&A]
The README mentions that one feature of phobos is
> Tools for maintaining your own model database

I've poked through the docs and don't see any explicit mention of this. If some documentation exists, could I be pointed to it?

Additionally, does phobos support stitching together multiple models into one greater definition? As in, I create a database of sub-components that I then want to stitch together later into a fully realized scene. I see in some of the CI documentation things relating to this, but I am not sure that is the correct mechanism to chase down for this functionality.

**Comment by m0rsch1** [comment]
Actually there is/was the Arrangement class for that specific purpose. I also want to use that feature but currently this does not seem to be implemented completely. See issue #367

**Comment by m0rsch1** [comment]
I have managed to get this working. I stitched together the submodels into an Arrangement, called assemble() to make it a single Robot and then exported it. My changes are in #373
